AGXUnity/BrickUnity/Factories/VisualFactory.cs

```diff
@@ -1,8 +1,9 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using UnityEngine;
 using System.IO;
 using UnityMeshImporter;
-using AGXUnity;
+using AGXUnity.Utils;
```

Comment:
Where these imports added? Didn't it work before?

Reply:
No, probably just something old that slipped through. Can remove it.